catalog/opensearch/catalog-opensearch-source/src/test/groovy/org/codice/ddf/opensearch/source/OpenSearchParserImplSpec.groovy

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.opensearch.source
+
+import org.apache.cxf.jaxrs.client.WebClient
+import spock.lang.Specification
+
+class OpenSearchParserImplSpec extends Specification {
+
+    private OpenSearchParser openSearchParser = new OpenSearchParserImpl()
+
+    private WebClient webClient = Mock(WebClient)
+
+    // {@link OpenSearchParser#populateContextual(WebClient, Map, List)} tests
```

Comment:
✏️ Extra comment.

Reply:
That was copied from the old file for consistency:
https://github.com/codice/ddf/blob/7e5f36cce802e4b7547ff6a3a880db6d1e6d850a/catalog/opensearch/catalog-opensearch-source/src/test/java/org/codice/ddf/opensearch/source/OpenSearchParserImplTest.java#L80